Dockerfile

```diff
@@ -125,15 +125,19 @@ WORKDIR /app
 COPY --from=build --chown=node /home/unlock/$BUILD_DIR/package.json package.json
 COPY --from=build --chown=node /home/unlock/$BUILD_DIR/yarn.lock yarn.lock
 
-# delete dev deps to prevent yarn from fetching them
+# delete dev deps (prevents yarn from fetching them)
+# add link: protocol workaround for yarn local links to work
 RUN apk add --no-cache --virtual .build-deps coreutils jq  \
-    && jq 'del(.devDependencies)' package.json > package.json.tmp \
+    && jq 'del(.devDependencies) | (.dependencies |= (with_entries(if .key == "@unlock-protocol/networks" then .key = "link:./packages/networks" else . end)))' package.json > package.json.tmp \
```

Comment:
Tu penses que ca peut valoir le coup de faire un truc plus generique car j'ai peur qu'on ait d'autres packages du meme genre?
Sinon, je me disais que les packages internes on a peut etre pas besoin de les prefix abec `@unlock-protocol`, wdyt?

Reply:
pour le prefixe, je pense que c'est bien que ce soit consistant et c'est utile pour yarn workspace. Par contre on peut ajouter `private: true` dans le package pour être sûr de ne pas le publier.

OK pour faire un truc plus générique i.e. remplacer par une regexp. Je me disais juste qu'on ferait ça le moment venu :P

Reply:
HA! noted :) Let's do it now because I worry we'll forget about this in 3 months and be confused! Thanks!

Reply:
If we decide to write a regex to replace `@unlock-protocol/` by `link:./packages` then we will run across the problem that `@unlock-protocol/unlock-js` or `@unlock-protocol/contract-abi-v-x` are not there. Of course the best will be for yarn to solve this